nginx/1.10/README.md

```diff
@@ -0,0 +1,92 @@
+# How to use NGINX on DC/OS
+
+[NGINX](https://www.nginx.com) is a high-performance HTTP server, reverse proxy, and an IMAP/POP3 proxy server. NGINX is known for its high performance, stability, rich feature set, simple configuration, and low resource consumption. DC/OS allows you to quickly configure, install and manage NGINX.
+
+- Estimated time for completion: 10 minutes
+- Target audience: Anyone interested in running a HTTP (proxy) server
+- Scope: Learn how to deploy NGINX service on DC/OS and to serve a static website
+
+## Prerequisites
+
+- A running DC/OS 1.10 cluster with at least 1 node having at least 1 CPUs and 1 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.10/cli/install/) installed.
+
+There are three ways to deploy NGINX container.
+1. [using Catalog](#deploy-nginx-using-catalog)
+2. [using CLI](#deploy-nginx-using-cli)
+
+## Deploy NGINX using Catalog
+
+Assuming you have a DC/OS cluster up and running, we install NGINX to serve up a static Web site, available via the GitHub repo [mesosphere/hello-nginx](https://github.com/mesosphere/hello-nginx).
+
+- Visit the DC/OS admin page, and click on `Catalog` on the left menu bar.
+- Search and select `nginx`
+![Catalog NGINX](img/catalog-nginx.png)
+- Click on `CONFIGURE` button
+- Enter "`hello-nginx-master`" in `CONTENTDIR` and "`https://github.com/mesosphere/hello-nginx/archive/master.zip`" in `CONTENTURL`
+![Catalog NGINX](img/nginx-config.png)
+- Click on `REVIEW AND DEPLOY`, verify values and click on `DOWNLOAD CONFIG.JSON` button from top right corner
+![Catalog NGINX](img/nginx-review.png)
+- Click on `DEPLOY` and wait
+- One popup should appear with `Success` message
+![Catalog NGINX](img/nginx-success.png)
+- Click on `Services` from left menu bar
+- there should be link button beside `nginx` service name, click on it or go to http://<MASTER-NODE-URL/ADDRESS>/service/nginx/
```

Comment:
s/there/There 
I will fix that during commit

Reply:
Oh, one typo. Thank you for noticing and reviewing.